packages/manager/src/views/SPAPropertyDetailPage/VirtualPath/components/DeleteVirtualPath.tsx

```diff
@@ -0,0 +1,273 @@
+                  value={field.value}
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  isDisabled
+                />
+              </FormGroup>
+            )}
+          />
+        </SplitItem>
+        <SplitItem isFilled style={{ width: '100%' }}>
+          <Controller
+            control={control}
+            name="identifier"
+            defaultValue={identifier}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup
+                label="Application Name"
+                isRequired
+                fieldId="identifier"
+                validated={error ? 'error' : 'default'}
+                helperTextInvalid={error?.message}
+              >
+                <TextInput
+                  isRequired
+                  placeholder="Add application name"
+                  type="text"
+                  id="identifier"
+                  value={field.value}
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  isDisabled
+                />
+              </FormGroup>
+            )}
+          />
+        </SplitItem>
+      </Split>
+      <Split hasGutter>
+        <SplitItem isFilled style={{ width: '100%' }}>
+          <Controller
+            control={control}
+            name="basePath"
+            defaultValue={data?.basePath}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup
+                label={
+                  <>
+                    Base Path
+                    <Tooltip content="Symlink virtualPath path should be /{spa-path}/{virtualPath-path}">
+                      <span>
+                        &nbsp; <InfoCircleIcon style={{ color: '#6A6E73' }} />
+                      </span>
+                    </Tooltip>
+                  </>
+                }
+                isRequired
+                fieldId="basePath"
+                validated={error ? 'error' : 'default'}
+                helperTextInvalid={error?.message}
+              >
+                <TextInput
+                  isRequired
+                  placeholder="Base path"
+                  type="text"
+                  id="basePath"
+                  value={field.value}
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  isDisabled
+                />
+              </FormGroup>
+            )}
+          />
+        </SplitItem>
+        <SplitItem isFilled style={{ width: '100%' }}>
+          <Controller
+            control={control}
+            name="virtualPath"
+            defaultValue={data.virtualPath}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup
+                label={
+                  <>
+                    Virtual Path
+                    <Tooltip content="Symlink basePath path should be /{spa-path}/{basePath-path}">
```

Comment:
Please remove the symlink from here, change it to the Virtual Path reference

Reply:
done

Reply:
done